src/Flysystem/FlysystemAssetStore.php

```diff
@@ -771,7 +772,7 @@ protected function cleanFilename($filename)
      * @param string $fileID
      * @return array
      */
-    protected function parseFileID($fileID)
+    public function parseFileID($fileID)
```

Comment:
We'll have a better way of doing this when the permalink job is merge.

Reply:
Okay, leaving a note in https://github.com/silverstripe/silverstripe-assets/issues/219